### ISSUE TITLE
Improve peewee support

### ISSUE THREAD

**Issue by cluster-master** [flask-admin/flask-admin#1070]
Hello Maintainers! Thank you for a good job!

Should improve peewee support with fixing that two issues described below:

1) Values that passed the `BooleanEqualFilter` and `BooleanNotEqualFilter` are '0' or '1', string values. 
But peewee doesn't support comparison for string and boolean values. Possible quick fix

```
def apply(self, query, value):
        return super(BooleanFilter, self).apply(query, value == '1' if isinstance(value, basestring) else value)
```

2) `contrib.peewee.ModelView` has two related methods - get_query and get_one. Moreover, get_one uses  standard peewee model instance method `.get()`, which can fail if `.select()` method of peewee model will be overloaded by user with some additional but needed filtering on some field. There may be more right to define `get_one` as:

```
def get_one(self, id):
        return self.get_query().where(getattr(self.model, self._primary_key) == id).get()
```

which allows us to overwrite just one method, `get_query`, in ModelView instead of changing two methods, in case of `.select()` overloaded.

**Comment by pawl** [comment]
1) was fixed recently in #1031, but the fix hasn't made it to a release yet

**Comment by petrus-jvrensburg** [comment]
Closing this now. But please re-open if  it's still a problem.
